src/Files.App/ViewModels/ToolbarViewModel.cs

```diff
@@ -1148,7 +1148,7 @@ public List<ListedItem> SelectedItems
 		public bool CanCopy => SelectedItems is not null && SelectedItems.Any();
 		public bool CanShare => SelectedItems is not null && SelectedItems.Any() && DataTransferManager.IsSupported() && !SelectedItems.Any(x => (x.IsShortcut && !x.IsLinkItem) || x.IsHiddenItem || (x.PrimaryItemAttribute == StorageItemTypes.Folder && !x.IsArchive));
 		public bool CanRename => SelectedItems is not null && SelectedItems.Count == 1;
-		public bool CanViewProperties => SelectedItems is not null && SelectedItems.Any();
+		public bool CanViewProperties => true;
```

Comment:
Does this enable the button on the home page?

Reply:
No, with this code that has not change. CanViewProperties becomes useless, to discuss if it is removed completely or left if necessary.
`IsEnabled="{x:Bind converters:MultiBooleanConverter.AndConvert(ViewModel.CanViewProperties, ViewModel.InstanceViewModel.IsPageTypeNotHome), Mode=OneWay, FallbackValue=False}"`